lib/Backend/GlobOpt.cpp

```diff
@@ -7184,13 +7212,29 @@ GlobOpt::SetSymStoreDirect(ValueInfo * valueInfo, Sym * sym)
     if (prevSymStore && prevSymStore->IsStackSym() &&
         prevSymStore->AsStackSym()->HasByteCodeRegSlot())
     {
-        Assert(this->blockData.changedSyms != nullptr);
-        this->blockData.changedSyms->Set(prevSymStore->m_id);
+        this->SetChangedSym(prevSymStore->m_id);
     }
     valueInfo->SetSymStore(sym);
 }
 
 void
+GlobOpt::SetChangedSym(SymID symId)
+{
+    GlobOptBlockData * globOptData = &this->currentBlock->globOptData;
```

Comment:
Seems the below comment in else statement serves the same purpose? globOptData in currentBlock is not initialized but that is handled by MergeCapturedValues?

---
In reply to: [72481209](https://github.com/Microsoft/ChakraCore/pull/1337#discussion_r72481209) [](ancestors = 72481209)